packages/central-server/src/database/models/Comment.js

```diff
@@ -1,10 +0,0 @@
-/**
```

Comment:
Deleted, as now that we no longer use the `isDeleteableViaApi` key, this model did nothing